src/main/java/com/github/dikhan/domain/TriggerIncident.java

```diff
@@ -1,106 +1,35 @@
 package com.github.dikhan.domain;
 
-import java.util.List;
 import java.util.Objects;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-
 public class TriggerIncident extends Incident {
-
-    @JsonProperty("client")
```

Comment:
The v2 api do accept the two fields you listed.  However, they are not documented, so perhaps we shouldn't support them.